spec/std/yaml/from_yaml_spec.cr

```diff
@@ -0,0 +1,397 @@
+require "spec"
+require "yaml"
+
+private class ObjectStub
+  def self.new(pp : YAML::PullParser)
```

Comment:
I'd recommend to use a more descriptive variable name, even in such a simple case.